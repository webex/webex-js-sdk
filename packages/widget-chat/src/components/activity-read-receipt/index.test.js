import React from 'react';
import {findRenderedDOMComponentWithClass, renderIntoDocument} from 'react-addons-test-utils';

import ActivityReadReceipt from '.';

describe(`ActivityReadReceipt component`, () => {
  let mockReadUsers;
  let component;
  beforeEach(() => {
    mockReadUsers = [{userId: `bernie`}, {userId: `adam`}];
    component = renderIntoDocument(
      <ActivityReadReceipt actors={mockReadUsers} />
    );
  });

  it(`is rendered properly`, () => {
    const renderedComponent = findRenderedDOMComponentWithClass(component, `activity-read-receipt`);
    expect(renderedComponent).toBeDefined();
  });

});
