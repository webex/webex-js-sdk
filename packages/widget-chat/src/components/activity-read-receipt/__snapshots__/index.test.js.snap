exports[`ActivityReadReceipt component renders properly 1`] = `
<div
  className="activity-read-receipt">
  <div
    className="avatar"
    style={undefined}>
    <span
      className={undefined}>
      B
    </span>
  </div>
  <div
    className="avatar"
    style={undefined}>
    <span
      className={undefined}>
      A
    </span>
  </div>
</div>
`;
